src/layouting/layouting.jl

```diff
@@ -8,7 +8,7 @@ end
 
 function attribute_per_char(string, attribute)
     n_words = 0
-    if attribute isa AbstractVector
+    if attribute isa Vector
```

Comment:
I'm not sure if this is fine... It now treats `Point` and `Vec` as constant per character, but also ranges which might be bad?

Reply:
I'll just switch to adding an extra branch for StaticArrays instead